app/src/plugins/dates.ts

```diff
@@ -14,19 +14,28 @@
  * limitations under the License.
  */
 
+
 const DAY_MS = 24 * 60 * 60 * 1000;
 
-export function renderDaysAgo(lastUpdated: number) {
+export function renderMonthsAgo(lastUpdated: number) {
   const now = new Date().getTime();
+  const monthsAgo = new Date().getMonth();
```

Comment:
This is not `monthsAgo` this is just a number 0-11:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth

You probably want to just see if daysAgo is more than 30 rather than deal with calendar months at all.

Reply:
This was helpful. Thank you, Sam